server/routes/initialAppointment/handlers/prisonCreated/timeServed/initialMeetingName.ts

```diff
@@ -1,8 +1,8 @@
 import { Request, Response } from 'express'
```

Comment:
yes in principle - it'd depend on how much extra complexity reusing it would bring in.